build/aclocal/ax_cxx_compile_stdcxx.m4

```diff
@@ -115,18 +129,17 @@ AC_DEFUN([AX_CXX_COMPILE_STDCXX], [dnl
     if test x$ac_success = xno; then
       AC_MSG_ERROR([*** A compiler with support for C++$1 language features is required.])
     fi
+  fi
+  if test x$ac_success = xno; then
+    HAVE_CXX$1=0
+    AC_MSG_NOTICE([No compiler with C++$1 support was found])
   else
-    if test x$ac_success = xno; then
-      HAVE_CXX$1=0
-      AC_MSG_NOTICE([No compiler with C++$1 support was found])
-    else
-      HAVE_CXX$1=1
-      AC_DEFINE(HAVE_CXX$1,1,
-                [define if the compiler supports basic C++$1 syntax])
-    fi
-
-    AC_SUBST(HAVE_CXX$1)
+    HAVE_CXX$1=1
+    AC_DEFINE(HAVE_CXX$1,1,
+              [define if the compiler supports basic C++$1 syntax])
   fi
+  AC_SUBST(HAVE_CXX$1)
+  m4_if([$1], [17], [AC_MSG_WARN([C++17 is not yet standardized, so the checks may change in incompatible ways anytime])])
```

Comment:
This is not true, it is standardized (2017/12: https://www.iso.org/standard/68564.html) but probably not tested a lot with wxwidgets.

Reply:
This is a 3rd party file, so we're not going to change without some really good reason. It could be worth submitting an update to [Autoconf archive](https://www.gnu.org/software/autoconf-archive/How-to-contribute.html#How-to-contribute) however.

Reply:
Also C++17 *is* [standardized](https://herbsutter.com/2017/09/06/c17-is-formally-approved/).

Reply:
OK, I understand, I did not see that. So we should leave it like that for our code.